sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -234,7 +234,7 @@ void flush() throws Exception {
               return RetryType.RETRY_ALL_OPERATIONS;
             },
             response -> {
-              LOG.info("Append to stream " + streamName + " succeded.");
+              LOG.info("Append to stream " + streamName + " succeeded.");
```

Comment:
I wonder if we should instead do something like this:

```suggestion
              LOG.info("Append to stream %s succeeded.", streamName);
```

Feel free to do or not.